test/tests/functional/pbs_sched_fifo.py

```diff
@@ -69,28 +69,25 @@ def test_sched_fifo(self):
 
         # Submit 3 jobs to workq2, 1 jobs to workq
         j1 = Job(TEST_USER, attrs={
-            ATTR_queue: "workq2"
+            ATTR_queue: "workq"
         })
         j2 = Job(TEST_USER, attrs={
             ATTR_queue: "workq2"
         })
         j3 = Job(TEST_USER, attrs={
-            ATTR_queue: "workq2"
-        })
-        j4 = Job(TEST_USER, attrs={
             ATTR_queue: "workq"
         })
 
         j_id1 = self.server.submit(j1)
         j_id2 = self.server.submit(j2)
         j_id3 = self.server.submit(j3)
-        j_id4 = self.server.submit(j4)
 
         # Turn scheduling on again
-        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+        self.server.manager(MGR_CMD_SET,
+                            SERVER,
+                            {'scheduling': 'True'})
```

Comment:
This probably can fit on 2 lines instead of 3.  It'd be more tidy.

Reply:
Done.